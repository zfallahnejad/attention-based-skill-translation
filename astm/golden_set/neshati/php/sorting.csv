476,sorting
50079,sorting
324584,sorting
2364629,sorting
1338292,sorting
2844319,sorting
142162,sorting
227665,sorting
507674,sorting
3859027,sorting
2637490,sorting
1634046,sorting
4833,sorting
1438393,sorting
1144035,sorting
1229023,sorting
1491895,sorting
113938,sorting
206403,sorting
722762,sorting
118068,sorting
