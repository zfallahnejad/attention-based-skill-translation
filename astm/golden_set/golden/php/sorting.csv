476,25,23,0.5208333333333334
50079,20,15,0.5714285714285714
324584,17,21,0.4473684210526316
2364629,14,10,0.5833333333333334
2844319,13,14,0.48148148148148145
1338292,13,12,0.52
142162,12,8,0.6
227665,11,6,0.6470588235294118
507674,11,9,0.55
3859027,10,1,0.9090909090909091
2637490,10,2,0.8333333333333334
1634046,9,9,0.5
4833,9,7,0.5625
1144035,8,8,0.5
1491895,8,8,0.5
1229023,8,0,1.0
1438393,8,3,0.7272727272727273
118068,7,8,0.4666666666666667
206403,7,2,0.7777777777777778
113938,7,10,0.4117647058823529
722762,7,3,0.7
